pages/k8s/cis-compliance.md

```diff
@@ -14,13 +14,78 @@ toc: False
 ---
 
 The **Center for Internet Security (CIS)** maintains a
-[Kubernetes benchmark][cis-benchmark] that is helpful to ensure clusters are
+[Kubernetes benchmark][cis-benchmark] that helps ensure clusters are
 deployed in accordance with security best practices. **Charmed Kubernetes**
 includes support for the [kube-bench][] utility, which reports how well a
-cluster complies with this benchmark. This page details how to run these
-tests.
+cluster complies with this benchmark. This page highlights compliance
+requirements as well as details on running the benchmark and analyzing test
+results.
 
-## Run kube-bench
+## Compliance Requirements
+
+**Charmed Kubernetes** 1.19 is compliant by default. Key configuration changes
+from previous releases include the following:
+
+### `kubernetes-master`
+
```

Comment:
worth putting a line and link  to the config docs? https://ubuntu.com/kubernetes/docs/charm-kubernetes-master#configuration

Reply:
hmmm, not sure about this.  i don't like the idea of linking directly to the #config anchor because not all these changes are user-configurable.  If we send people to a blob of charm config, I think they might assume something like `profiling` can be turned on or off; it cannot as it is hard coded in the charm.

I like 2 things better:
1. link to https://ubuntu.com/kubernetes/docs/charm-kubernetes-master (no anchor) as a convenience for people to see what charm these changes affect
1. link the options that *can* be configured to the appropriate anchor:
    - `authentication.mode` could link to k8s-master [configuration](https://ubuntu.com/kubernetes/docs/charm-kubernetes-master#configuration)
    - admission control plugins could link to k8s-master [admission-controls](https://ubuntu.com/kubernetes/docs/charm-kubernetes-master#admission-controls).

Whatcha think?